.github/workflows/empty-worksapce-smoke-test-on-minikube.yaml

```
@@ -0,0 +1,88 @@
+#
+# Copyright (c) 2019-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+name: Empty workspace smoke test
```

This an API test right? I think we agree to call smoke test the graphical test 
```suggestion
name: Empty workspace API test
```